main.js

```diff
@@ -0,0 +1,235 @@
+//variable assignemnt, move tracker, and game status
+let cell0 = document.getElementById('cell-0');
+let cell1 = document.getElementById('cell-1');
+let cell2 = document.getElementById('cell-2');
+let cell3 = document.getElementById('cell-3');
+let cell4 = document.getElementById('cell-4');
+let cell5 = document.getElementById('cell-5');
+let cell6 = document.getElementById('cell-6');
+let cell7 = document.getElementById('cell-7');
+let cell8 = document.getElementById('cell-8');
+let cellNumber;
+let clock = document.getElementById('time');
+let startButton = document.getElementById('start');
+let resetButton = document.getElementById('reset');
+let player1Button = document.getElementById('p1Button');
+let player2Button = document.getElementById('p2Button');
+let pVPButton = document.getElementById('PvP');
+let pVCButton = document.getElementById('PvC');
+let playerTurn = document.getElementById('playerStatus');
+let playerXName = document.getElementById('playerX');
+let playerOName = document.getElementById('playerO');
+let currentPlayer = playerXName.textContent;
+let seconds = Number(clock.textContent);
```

Comment:
It would be better to just initialize this as the number `0` since this value will be set to that anyway, and only changes because our JavaScript changes it (it doesn't query the DOM again after the initial value is set), but it would make it clearer what this value is to start with.